libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/accounts/DisabledSku.kt

```diff
@@ -0,0 +1,17 @@
+package com.mapbox.services.android.navigation.v5.internal.accounts
+
+import com.mapbox.android.accounts.navigation.sku.v1.SkuGenerator
+
+// TODO: Remove this class when SKU is ready to be released publicly
```

Comment:
Should we remove this class as we're going to merge this PR into `master` and start testing on the field? 👀 https://github.com/mapbox/mapbox-navigation-android/pull/2063/files#r342186221